Backendless/Service/PersistenceService.cs

```diff
@@ -442,7 +442,7 @@ public T First<T>( DataQueryBuilder queryBuilder )
       if( relations == null )
         relations = new List<String>();
 
-      int relationsDepth = queryBuilder.GetRelationsDepth();
+      int? relationsDepth = queryBuilder.GetRelationsDepth();

```

Comment:
for this line and below - when the value is null, it should not be sent to the server